makefile

```diff
@@ -152,14 +152,29 @@ build-mpi: $(MPI_TARGETS)
 	@echo '--- boost mpi bindings built ---'
 
 .PHONY: build
-build: bin/stanc$(EXE) bin/stansummary$(EXE) bin/print$(EXE) bin/diagnose$(EXE) $(LIBSUNDIALS) $(MPI_TARGETS) $(CMDSTAN_MAIN_O)
-	@echo ''
+build: bin/stanc$(EXE) bin/stansummary$(EXE) bin/print$(EXE) bin/diagnose$(EXE) $(LIBSUNDIALS) $(MPI_TARGETS) $(TBB_TARGETS) $(CMDSTAN_MAIN_O)
+	@echo ''
+	@if [ "$(OS)" == "Windows_NT" ]; then \
+		echo 'NOTE: Please add $(TBB_BIN_ABSOLUTE_PATH) to your PATH variable.'; \
+		echo 'You may call'; \
+		echo ''; \
+		echo 'mingw32-make install-tbb'; \
+		echo ''; \
+		echo 'to automatically update your user configuration.'; \
+	fi
 	@echo '--- CmdStan v$(CMDSTAN_VERSION) built ---'
 
 ifeq ($(CXX_TYPE),clang)
 build: $(STAN)src/stan/model/model_header.hpp.gch
 endif
 
+.PHONY: install-tbb
+install-tbb: $(TBB_TARGETS)
+	@if ! [ "$(OS)" == "Windows_NT" ]; then \
+		exit 0; \
```

Comment:
And this to 

```
ifeq ($(OS),Windows_NT)
		exit 0;
endif
```

Sorry I missed this one before.

Reply:
ok. much nicer.